### PR TITLE
Initial commit of schema definition for invalid updates.

### PR DESCRIPTION
**Pull request by andrewpollock**
This is my first foray into writing a JSON schema, and I've basically done it using https://www.jsonschema.net/app/schemas/231529 so let me know if you see any issues with it (including things like `$id`)

Contributes to #771

### CODE REVIEW

**Review by michaelkedar — COMMENTED** [review]
A few comments as I read up on JSON schema :)

File: docs/invalid_update.schema.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    "examples": [{
+        "invalid_records": [{
+            "id": "GSD-2021-1001229",
+            "reason": "INVALID_JSON"
+        },
+        {
+            "id": "GSD-2022-1003535",
+            "reason": "DELETED"
+        }]
+    }]
```
[the remark this answers is not in the document]
Comment (andrewpollock) [reply docs/invalid_update.schema.json:71]: Agreed, trimmed.

File: docs/invalid_update.schema.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        "invalid_records"
+    ],
+    "properties": {
+        "invalid_records": {
+            "type": "array",
+            "default": [],
+            "title": "The invalid_records Schema",
+            "items": {
+                "type": "object",
+                "title": "A Schema",
```
[the remark this answers is not in the document]
Comment (andrewpollock) [reply docs/invalid_update.schema.json]: Ah yes, yes it is. Improved.

File: docs/invalid_update.schema.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                        ]
+                    },
+                    "reason": {
+                        "type": "string",
+                        "title": "The reason Schema",
+                        "examples": [
+                            "INVALID_JSON",
+                            "DELETED"
+                        ]
+                    }
```
[the remark this answers is not in the document]
Comment (andrewpollock) [reply docs/invalid_update.schema.json]: Oooh good call, I didn't realise you could do that.

File: docs/invalid_update.schema.json [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    "title": "OSV.dev import status",
+    "description": "A schema for describing an OSV.dev import failure of an OSV record.",
+    "required": [
+        "invalid_records"
+    ],
+    "properties": {
+        "invalid_records": {
+            "type": "array",
+            "default": [],
+            "title": "The invalid_records Schema",
```
[the remark this answers is not in the document]
Comment (andrewpollock) [reply docs/invalid_update.schema.json]: I've improved on them.

**Review by oliverchang — APPROVED** [review]

**Pull request closed without merging**